labrad/client.py

```diff
@@ -66,7 +66,7 @@ def __call__(self, *args, **kw):
 
     def future(self, *args, **kw):
         wrap = kw.pop('wrap', True)
-        tag = kw.pop('tag', None) or self.accepts
+        tag = kw.pop('tag', None) or self.accepts_type
```

Comment:
This can be `tag = kw.pop('tag', self.accepts_type)` unless you mean to take advantage of the short circuit or to avoid evaluating the property.